From efee640716ed4f3273ea3e24e0b0b0d9b792433a Mon Sep 17 00:00:00 2001
From: Paul Spooren <mail@aparcar.org>
Date: Thu, 20 May 2021 12:08:10 +0200
Subject: [PATCH 51/61] busybox: mention SRV support in help message

The SRV was added some time ago and should be mentioned in the short
help message to avoid confusion about missing features.

Signed-off-by: Paul Spooren <mail@aparcar.org>
---
 .../540-nslookup-mention-QUERY_TYPE-SRV.patch | 30 +++++++++++++++++++
 1 file changed, 30 insertions(+)
 create mode 100644 package/utils/busybox/patches/540-nslookup-mention-QUERY_TYPE-SRV.patch

diff --git a/package/utils/busybox/patches/540-nslookup-mention-QUERY_TYPE-SRV.patch b/package/utils/busybox/patches/540-nslookup-mention-QUERY_TYPE-SRV.patch
new file mode 100644
index 0000000000..ac9afa34ef
--- /dev/null
+++ b/package/utils/busybox/patches/540-nslookup-mention-QUERY_TYPE-SRV.patch
@@ -0,0 +1,30 @@
+From a0136f24f788e8bfc4ab74a647b27e115a25d9fb Mon Sep 17 00:00:00 2001
+From: Paul Spooren <mail@aparcar.org>
+Date: Wed, 19 May 2021 00:55:21 +0200
+Subject: [PATCH] nslookup: mention QUERY_TYPE SRV
+
+SRV lookups are supported since "6b4960155 nslookup: implement support
+for SRV records" and should therefore be mentioned as a possible
+QUERY_TYPE in the help message.
+
+Signed-off-by: Paul Spooren <mail@aparcar.org>
+---
+ networking/nslookup.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/networking/nslookup.c b/networking/nslookup.c
+index dda22de0e..2ae8d391c 100644
+--- a/networking/nslookup.c
++++ b/networking/nslookup.c
+@@ -25,7 +25,7 @@
+ //usage:#define nslookup_full_usage "\n\n"
+ //usage:       "Query DNS about HOST"
+ //usage:       IF_FEATURE_NSLOOKUP_BIG("\n")
+-//usage:       IF_FEATURE_NSLOOKUP_BIG("\nQUERY_TYPE: soa,ns,a,"IF_FEATURE_IPV6("aaaa,")"cname,mx,txt,ptr,any")
++//usage:       IF_FEATURE_NSLOOKUP_BIG("\nQUERY_TYPE: soa,ns,a,"IF_FEATURE_IPV6("aaaa,")"cname,mx,txt,ptr,srv,any")
+ //usage:#define nslookup_example_usage
+ //usage:       "$ nslookup localhost\n"
+ //usage:       "Server:     default\n"
+-- 
+2.30.2
+
-- 
2.30.2

